Towny/src/main/java/com/palmergames/bukkit/towny/db/TownySQLSource.java

```diff
@@ -1005,6 +1005,8 @@ private boolean loadTown(ResultSet rs) {
 			town.setAllowedToWar(rs.getBoolean("allowedToWar"));
 			town.setJoinedNationAt(rs.getLong("joinedNationAt"));
 			town.setMovedHomeBlockAt(rs.getLong("movedHomeBlockAt"));
+			town.setForSale(rs.getBoolean("forSale"));
```

Comment:
These two new options will not appear and I believe it will be safer to do what is being done for the maxPercentTaxAmount:
line = rs.getBoolean("forSale")
if (line != null)
    town.setForSale(line);

This should be done for both forSale and forSalePrice.